google-cloud-storage/lib/google/cloud/storage/version.rb

```diff
@@ -16,7 +16,7 @@
 module Google
   module Cloud
     module Storage
-      VERSION = "1.52.0".freeze
```

Comment:
Please do not update the version manually. This is read and updated during the release process, and modifying it manually will confuse the releaser.